### PR TITLE
mEmiFiEd cOdE aNd aDdEd aLpHa10

### PR DESCRIPTION
**Pull request by jojoe77777**
sUrPrIsInGlY, tHiS wOrKs

### CODE REVIEW

File: src/dktapps/sPonGeBoBmEMe/Main.php [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-	public function onPlayerChat(PlayerChatEvent $ev){
-		$message = str_split($ev->getMessage());
-		foreach($message as $k => $char){
-			$char = strtolower($char);
-			if(mt_rand(0, 1) === 1){
-				$char = strtoupper($char);
+	pUbLIc FuNcTIOn OnpLaYErcHAt(pLAyErcHaTeVeNT $eV){
+		$mEssAgE = StR_SPlIT($eV->gEtmESsAgE());
+		foReAcH($mEssAgE aS $K => $ChAR){
+			$char = StRtOLoWEr($ChAR);
```
Comment (dktapps) [inline src/dktapps/sPonGeBoBmEMe/Main.php]: missed one here, `$char` is undefined

**Comment by dktapps** [comment]
`Main.php` shOulD prObaBly Be rEnaMed to maTcH thE clAss naMe, or aUtoLoadInG migHT BrEAk iN ThE FutuRE.

**Comment by jojoe77777** [comment]
uGh i dID tHaT bUt GiT iGnOrEs iT

**Comment by dktapps** [comment]
tHat'S IncOnveNienT. I'lL FiX iT LatEr theN.

**Pull request merged**